mule-extensions-api/src/main/java/org/mule/runtime/extension/api/util/XmlModelUtils.java

```diff
@@ -73,34 +73,34 @@
   /**
    * Takes a set of parameters extracted from the extension and generates a {@link XmlDslModel}.
    *
-   * @param extensionNamespace namespace of the extension. If {@link Optional#empty()} or empty string, then it will default using the {@code extensionName}.
-   * @param extensionNamespaceLocation namespace location of the extension. If {@link Optional#empty()} or empty string, then it will default using a generated namespace.
+   * @param extensionPrefix namespace of the extension. If {@link Optional#empty()} or empty string, then it will default using the {@code extensionName}.
+   * @param extensionNamespace namespace location of the extension. If {@link Optional#empty()} or empty string, then it will default using a generated namespace.
    * @param extensionName name of the extension, cannot be null.
    * @param extensionVersion version of the extension, cannot be null.
    * @return a wellformed {@link XmlDslModel}
    */
-  public static XmlDslModel createXmlLanguageModel(Optional<String> extensionNamespace,
-                                                   Optional<String> extensionNamespaceLocation, String extensionName,
+  public static XmlDslModel createXmlLanguageModel(Optional<String> extensionPrefix,
```

Comment:
rename to simply prefix and namespace... the `extension namespace` is a concept that came after this was originally coded and represent a different thing